include/rootston/seat.h

```diff
@@ -23,6 +26,10 @@ struct roots_seat {
 	struct wl_list link;
 	struct wl_list drag_icons;
 
+	// coordinates of the first touch point if it exists
+	int32_t touch_id;
+	double touch_x, touch_y;
```

Comment:
What happens if there are multiple touch points? What are the consequences of only considering the first?

Reply:
The intent is to consider only the first touch point for window manager actions. So moving around a window with touch will only use the first touch point and ignore the rest. It would be pretty cool to move two windows around at the same time with two touch points, but that would require a lot of work.

Reply:
Okay, let's open an issue for this feature then!